### ISSUE TITLE
Intercooler on an Angular app

### ISSUE THREAD

**Issue by guadare** [LeadDyno/intercooler-js#207]
Hi, 

I have an Angular App and I try to use intercooler on this app. It seem that this combination (Angular **>** Intercooler) is not possible because  the intercooler events are not executed. I tried with **ic-post-to**, **ic-get-from** and i**c-poll** attributes. I tested intercooler in a plain HTML and it works fine with the same backend but I can't get that works on my Angular app. 

There is some reason why it is not possible to use Intercooler in an Angular app?

Thank you.

**Comment by carsongross** [comment]
My guess is that Angular is mutating the DOM after the HTML is loaded.
Unfortunately that means you are going to have to call

  Intercooler.processNodes(elt)

on whatever content is mutated, which might be really tough with Angular
since it is constantly mutating the DOM.  :(

Is it possible to make the sub-section that you want to use intercooler or
non-Angular code?

Cheers,
Carson

On June 8, 2017 at 5:38:51 AM, guadare (notifications@github.com) wrote:

Hi,

I have an Angular App and I try to use intercooler on this app. It seem
that this combination (Angular *>* Intercooler) is not possible because the
intercooler events are not executed. I tried with *ic-post-to*,
*ic-get-from* and i*c-poll* attributes. I tested intercooler in a plain
HTML and it works fine with the same backend but I can't get that works on
my Angular app.

There is some reason why it is not possible to use Intercooler in an
Angular app?

Thank you.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/LeadDyno/intercooler-js/issues/207>, or mute the thread
